src/test_workflow/integ_test/distribution.py

```diff
@@ -47,8 +47,9 @@ def start_cmd(self) -> str:
         """
         pass
 
+    @abstractmethod
     def uninstall(self) -> None:
         """
-        Allow distribution that is not 'tar' to do proper cleanup
+        Allow distribution to do proper cleanup
```

Comment:
To cleanup. Proper is unnecessary filler.

Reply:
Will change later.